packages/financial-templates-lib/src/logger/Transports.js

```diff
@@ -18,15 +18,15 @@ function createTransports(transportsConfig = {}) {
   // If the logger is running in serverless mode then add the GCP winston transport and console transport.
   if ((transportsConfig.environment ?? process.env.ENVIRONMENT) == "serverless") {
     const { LoggingWinston } = require("@google-cloud/logging-winston");
-    require("@google-cloud/trace-agent").start();
+    if (!require("@google-cloud/trace-agent").get().enabled) require("@google-cloud/trace-agent").start();
```

Comment:
this library is imported mulitple times, is there a reason you dont just import once?

Reply:
i dont see an easy way to do that, unless its instanced at the top level and passed in as a dependency, but its not worth the effort.